src/test/SavingsDai-integration.t.sol

```diff
@@ -731,7 +766,7 @@ contract SavingsDaiIntegrationTest is DSSTest {
         if (to == address(0) || to == address(token)) return;
 
         uint256 pie = mintAmount * RAY / pot.chi();
-        burnAmount = bound(burnAmount, pie + 1, type(uint256).max);
+        burnAmount = bound(burnAmount, pie + 1, type(uint256).max / pot.chi());
```

Comment:
Just to verify I understood the problem correctly, it was basically finding cases that the burn amount actually overflow multiplying by `chi`, then wasn't reverting by the expected case.